capture/screenshot.js

```diff
@@ -1,15 +1,15 @@
-import sendRequest from '../send/request.js'
+import sendRequest from '../utils/request.js'
 import request from 'request'
 import config from '../config/index.js'
 import fs from 'fs'
 
-const dir = 'tmp'
+const DIR = 'tmp'
```

Comment:
I just wondering, why dont const DIR is define on `/config` ?
I thought config directory is for define any constants

CMIIW

Reply:
i see, that your right @setiadijoe

Reply:
thanks, i was update 0d28a51 :smile: